packages/@aws-cdk-testing/framework-integ/test/aws-synthetics/test/integ.canary-auto-delete-lambda.ts

```diff
@@ -54,8 +54,21 @@ class TestStack extends Stack {
 
 const app = new App();
 
-new IntegTest(app, 'cdk-integ-synthetics-canary-auto-delete', {
+const integTest = new IntegTest(app, 'cdk-integ-synthetics-canary-auto-delete', {
   testCases: [new TestStack(app, 'cdk-synthetics-canary-auto-delete')],
   diffAssets: true,
-  stackUpdateWorkflow: false, // will error because this stack has a cr that deletes its own resources
```

Comment:
Can we keep the comment here? My first question when I saw this was, "why are we we disabling the update workflow for this integ test". The context provided by the comment is useful.

Reply:
Will add it back. 👍🏻